ansible/roles-infra/infra-ec2-template-generate/templates/cloud_template.j2

```diff
@@ -78,6 +78,7 @@ Resources:
     Type: "AWS::EC2::SecurityGroup"
     Properties:
       GroupDescription: Host
+      GroupName: "{{security_group['name']}}"
```

Comment:
Commented on the issue 

> Can you elaborate why having a random hash for GroupName is an issue?

Reply:
@fridim since the hash is not a derivative of the GUID or any other identifier, there is no way to publish in the user details the user the name or id of the security group to use for subsequent automation in the open environment. Unlike other AWS resources such as an instance, keypair, or subnet, the "Name" tag does not actually represent the Name of the object so it cannot be referenced with only the tag.